src/clusterfuzz/_internal/bot/fuzzers/utils.py

```diff
@@ -79,6 +80,7 @@ def is_fuzz_target_local(file_path, file_handle=None):
   for pattern in FUZZ_TARGET_SEARCH_BYTES:
     # TODO(metzman): Bound this call so we don't read forever if something went
     # wrong.
+    local_file_handle.seek(0)
```

Comment:
Thanks for catching this!

Reply:
No probs, I think I introduced the bug by adding the loop in the first place.